### DISCUSSION TITLE
Add unbound option to Adguard Home installation

### DISCUSSION THREAD

**Discussion by ronlut** [tteck/Proxmox#2489] [Ideas]
Pretty straightforward, like done in #2425 for Pihole.
I would happily work on it myself, if it sounds like a good idea.
I just need some structuring guidelines - I think I would extract the unbound installation and configuration to a single script, where would you place it and how to invoke it from an `X-install.sh` script? Does that make sense?
LMKWYT, Thanks!

**Comment by thiagohflima** [comment]
Up! 👍

**Comment by purple-emily** [comment]
This is exactly what I was looking for!

**Comment by MatKlein** [comment]
I just set that up on two nodes with Docker, combined with keepalived for high availability, but have issues with IPv6. Would also be happy about an option with a helper script.

**Comment by gigi2006** [comment]
realy good idea, plaese update this :)

**Comment by ianalexis** [comment]
UPalala
